### PR TITLE
Sndio defaults

### PR DESCRIPTION
**Pull request by Kistelini**
All other packages offer sndio support as an option which is enabled by default or via a sub package.

### CODE REVIEW

File: srcpkgs/vlc/template [hunk 1]
```diff
@@ -29,19 +29,19 @@ makedepends="
...
  $(vopt_if vdpau libvdpau-devel) $(vopt_if x265 x265-devel)
- $(vopt_if svg librsvg-devel)"
+ $(vopt_if svg librsvg-devel) $(vopt_if sndio sndio-devel)"
 depends="freefont-ttf hicolor-icon-theme desktop-file-utils"
 
 # Package build options
-build_options="lua notify opengl qt smb svg v4l2 vaapi vdpau x265"
-build_options_default="lua smb v4l2 x265"
+build_options="lua notify opengl qt smb svg v4l2 vaapi vdpau x265 sndio"
+build_options_default="lua smb v4l2 x265 sndio"
```
Comment (maxice8) [inline srcpkgs/vlc/template]: neeeds revbump

File: srcpkgs/lmms/template [hunk 2]
```diff
@@ -25,6 +25,9 @@ checksum="2efba2eda8ec3030ff6d446fd7bbe2245b38472f60c622b9e41053b1981fd592
  06159c71a5eeba5662bfb309d215e733f310c2699c8c1de83297d32d13a510bc
  9bc0013a0bf8c01540821e84fef0cef067f1d857a01ceceac36f83bd6fedc106"
 
+build_options="sndio"
+build_options_default="sndio"
```
Comment (maxice8) [inline srcpkgs/lmms/template]: needs revbump

**Comment by maxice8** [comment]
I'd rather have individual PRs

**Comment by Kistelini** [comment]
See https://github.com/void-linux/void-packages/pull/7930 https://github.com/void-linux/void-packages/pull/7929 https://github.com/void-linux/void-packages/pull/7928 https://github.com/void-linux/void-packages/pull/7926 https://github.com/void-linux/void-packages/pull/7925

**Pull request closed without merging**